server/services/reportedAdjudicationsService.ts

```diff
@@ -298,6 +298,10 @@ export default class ReportedAdjudicationsService {
     const friendlyName =
       (prisonerResult && convertToTitleCase(`${prisonerResult.firstName} ${prisonerResult.lastName}`)) || ''
     const reportingOfficer = (reporterName && convertToTitleCase(reporterName)) || ''
+    const nextSheduledHearingDate =
```

Comment:
Can we rename the variable something like `lastScheduledHearingDate` ? I think naming it `nextScheduledHearingDate` is confusing as it might not be the next one, it's the final one, isn't it?